projects/collections/collection.js

```diff
@@ -3,10 +3,37 @@ Projects = new Mongo.Collection('projects');
 //Add search index with easy search
 ProjectsIndex  = new EasySearch.Index({
     collection: Projects,
-    fields: ['name','tags','postalAdress'],
-    engine: new EasySearch.Minimongo()
-  });
-  
+    fields: ['name','tags'],
+    engine: new EasySearch.Minimongo({
+      selector: function (searchObject, options, aggregation) {
+        // Set selector to Default
+        let selector = this.defaultConfiguration().selector(searchObject, options, aggregation);
+
+        // Reactively gets the queryparams to sort the /projects page
+        let category = FlowRouter.getQueryParam("cat");
+        let location = FlowRouter.getQueryParam("loc");
+
+        // If there is a category querayparam, add it to the selector
+        if(category != undefined && category.length > 0){
+          selector.problemCategories = category;
+        };
+
+        // If there is a location queryparam, add it to the selector
+        if(location != undefined && location.length > 0){
+
+        // _. extend below is not EasySearch standard syntax for adding value to selector
+        // Dot notation for nested objects does not work in EasySearch.
+        // See: https://docs.mongodb.org/v2.2/core/read-operations/#subdocuments
+        // This has to be used whenever easysearch should filter on a nested...
+        // objects property.
+        _.extend(selector, {"postalAddress.country": location});
```

Comment:
Check indentation here. Nested blocks should be indented by one level (two spaces).